src/tools/debugger/FrontendConnector.java

```diff
@@ -207,18 +209,53 @@ public void awaitClient() {
   private static Map<SFarReference, String> createActorMap(
```

Comment:
In the Other Branch both methods are already gone, but this branch still needs them.
createActorMap contains actors since the last breakpoint
creatObjectMap is a map of all farrefs (necessary to map senders/receivers to ids)